### PR TITLE
fix minor typos

### PR DESCRIPTION
**Pull request by RayAsh37**
I have fixed some very minor but significant typos in the docs. Please review.

### CODE REVIEW

**Review by ronak-pal1 — APPROVED** [review]

File: docs/templates/pagerduty-integration.md [hunk 1]
```diff
@@ -76,7 +76,7 @@ If you need help with this integration, please contact ***[support@fiberplane.co
 1. Go to the **Incidents** screen in PagerDuty and click **+ New Incident**
 2. Select an **Impacted Service** and add a **Title** to the incident and optionally select an **Urgency, Incident Priority** and **Description.**
 3. Click **Create Incident**
-4. Click **! Acknowledge**
+4. Click **Acknowledge**
```
Comment (keturiosakys) [inline docs/templates/pagerduty-integration.md:79]: The button on the Pagerduty interface that this line refers to has an exclamation mark so this change is not necessary:

```suggestion
4. Click **! Acknowledge**
```

**Review by keturiosakys — CHANGES_REQUESTED** [review]
Thanks for your first contribution @RayAsh37!

Requesting one small change but looks good otherwise.

File: docs/templates/pagerduty-integration.md (hunk 1 above)
Comment (RayAsh37) [reply docs/templates/pagerduty-integration.md]: Done!

**Comment by RayAsh37** [comment]
@keturiosakys 

I have done the required changes. Please review.

**Review by keturiosakys — APPROVED** [review]

**Pull request merged**